packages/klevu-core/src/models/KlevuFetchResponse.ts

```diff
@@ -32,7 +32,24 @@ export type KlevuFetchQueryResult = KlevuQueryResult &
      */
     functionParams?: KlevuFecthFunctionParams
   }
-
+/**
+ * Fetch query results
+ */
+export type KlevuAnnotations = {
+        /** Api key that was used  */
+        apiKey?: string,
```

Comment:
Are all of the keys really optional in the result? Even the apiKey?

Reply:
acording to api docs provided , yes keys might not be returned